pkg/cli/build.go

```diff
@@ -0,0 +1,200 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/maps"
+
+	"github.com/wolfi-dev/wolfictl/pkg/dag"
+)
+
+func Build() *cobra.Command {
+	var archs []string
+	var dir, pipelineDir string
+	var jobs int
+	var dryrun bool
+	cmd := &cobra.Command{
+		Use:           "build",
+		SilenceErrors: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
+			if jobs == 0 {
+				jobs = runtime.GOMAXPROCS(0)
+			}
+			jobch := make(chan struct{}, jobs)
+
+			newTask := func(pkg string) *task {
+				return &task{
+					pkg:         pkg,
+					dir:         dir,
+					pipelineDir: pipelineDir,
+					stdout:      cmd.OutOrStdout(),
+					stderr:      cmd.ErrOrStderr(),
+					archs:       archs,
+					dryrun:      dryrun,
+					done:        make(chan struct{}),
+					deps:        map[string]chan struct{}{},
+					jobch:       jobch,
+				}
+			}
+
+			if len(args) == 1 {
+				// Build only this one package.
```

Comment:
What if dependencies need to be rebuilt? 🤔

Reply:
That's going to be a followup PR, where I add `--buildworld` (whether or not to use deps from the package repo or build them ourselves) and `--todo-needs-a-name` (whether or not to build dependent packages)

If args are omitted we'll build everything.